documentation/cloud-manager-guide/admin/user-management/create-user.md

```diff
@@ -0,0 +1,82 @@
+---
+keywords:
+title: Create a User
+description: Create new users
+---
+# Create a User
+
+This guide outlines the required steps to create new users and assign roles in Environment Operations Center. What a user can view and which actions they can perform are dependent on their assigned role. User roles include Tenant Administrator, Environment Administrator, and Environment User. For details on permissions for each role, see the [role based permissions](../role-based-permission/role-based-permissions.md) guide.
+
+## Getting started
+
+To create a new user, select the **Create User** button on the *Users* screen.
+
+![image description](images/create-button.png)
+
+This takes you to the *Create User* form that contains all the required input fields to create a new user. Within the form you will provide the user's details and assign their role and environments.
```

Comment:
Remove "This take you to" and just start with "The *Create User* form contains all of the required fields..."